photo/migrations/0009_alter_picture_picture_path.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2 on 2023-09-18 15:56
+
+from django.db import migrations, models
+import photo.storages_backend
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("photo", "0008_alter_user_profile_picture"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="picture",
+            name="picture_path",
+            field=models.FileField(
+                storage=photo.storages_backend.PublicMediaStorage(), upload_to=""
```

Comment:
shouldn't we create a folder for this files, make use of the upload_to argument?